pkg/ccl/importccl/read_import_base.go

```diff
@@ -236,11 +236,11 @@ func readInputFiles(
 				})
 
 				if err := grp.Wait(); err != nil {
-					return errors.Wrap(err, dataFile)
```

Comment:
I don't like that this a requirement in our codebase, to be honest... does this really feel ergonomic or useful to you as a lint rule? I won't block this PR (thanks very much for doing this toil) but I do feel this deserves some questioning.

Reply:
I think this rule is trying to make sure we don't pass off PII information as what the error library expects is a simple hardcoded message. The case below is more relevant - `errors.Wrap(err, typedExpr.String())`. Do we want the string representation of the typedExpr to be treated as non-PII?

Reply:
_does_ the error library expect the argument to `Wrap` (not wrapf), to be a simple hardcoded message?

Reply:
> does the error library expect the argument to Wrap (not wrapf), to be a simple hardcoded message

No it does not. Jordan is right. I'll explain more in a comment under.